### PR TITLE
Stub out RWD

### PR DESCRIPTION
**Pull request by lewfish**
This PR stubs out RWD. To test, click on 'Hill Slope' in the 'Delineate Watershed' dropdown and then click a point on the map. This will make a spinner visible next to 'Hill Slope', and will poll the backend, which should return a diamond shaped polygon. The other menu items should work as before (creating random polygons on the frontend). If you disable your network connection before clicking 'Hill Slope', you should see an error icon instead of a spinner, which will be removed if you make another selection from the dropdown. 

I'm not sure how to display the actual error message. We could associate a tooltip with the error icon, but that might not be obvious enough to the user.

Connects #1058

### CODE REVIEW

File: src/mmw/js/src/draw/views.js [hunk 1]
```diff
@@ -338,24 +340,77 @@ var PlaceMarkerView = Marionette.ItemView.extend({
...
+                            point.geometry.coordinates[1],
+                            point.geometry.coordinates[0]
+                        ])}
+                    };
+
+                    self.rwdTaskModel.start(taskHelper);
+                    return deferred;
+                } else {
                     shape = turfRandom('polygons', 1, {
                         max_vertcies: 50,
```
Comment (kdeloach) [inline src/mmw/js/src/draw/views.js]: Unrelated, but I just noticed there's a typo on this line `max_vertcies`. Probably do not need turf-random anymore though.

File: src/mmw/js/src/modeling/models.js [hunk 2]
```diff
@@ -40,6 +40,16 @@ var Tr55TaskModel = coreModels.TaskModel.extend({
...
 
+var RwdTaskModel = coreModels.TaskModel.extend({
+    defaults: _.extend(
+        {
+            taskName: 'rwd',
+            taskType: 'modeling'
+        },
+        coreModels.TaskModel.prototype.defaults
+    )
+});
```
Comment (kdeloach) [inline src/mmw/js/src/modeling/models.js]: Looks like this model is not exported. It's also defined in `src/mmw/js/src/draw/models.js`.

File: src/mmw/js/src/draw/views.js (hunk 1 above)
Comment (kdeloach) [inline src/mmw/js/src/draw/views.js]: Oh right, there's other delineation models that still use a random polygon. Ignore that remark.

**Comment by kdeloach** [comment]
Very cool. One question: Should RWD be activated by clicking the "Stream Network" option instead of the "Hill Slope" option?

**Comment by kdeloach** [comment]
+1 Tested

**Comment by mmcfarland** [comment]
Here's a brief description I got for what each of the 3 "Delineate Watershed" options will/should do:

* Trace Flow Down:: Interactive visualization of blue line to the ocean
* Hill Slope:: Just local operation of Stream Network not snapped to blue line
* Stream Network:: Rapid Watershed Deliniation snap to blue line and compute contributing area

**Comment by lewfish** [comment]
@azaveaci rebuild

**Pull request merged**